packages/account/src/Sections/Profile/PhoneVerification/session-timeout-modal.tsx

```diff
@@ -7,21 +7,33 @@ import { useDevice } from '@deriv-com/ui';
 import { observer, useStore } from '@deriv/stores';
 import { useEffect } from 'react';
 
-const SessionTimeoutModal = observer(() => {
+const SessionTimeoutModal = observer(({ is_at_otp_verification }: { is_at_otp_verification: boolean }) => {
     const { isMobile } = useDevice();
     const history = useHistory();
     const { localize } = useTranslations();
     const { should_show_session_timeout_modal } = usePhoneNumberVerificationSessionTimer();
     const { ui } = useStore();
-    const { is_phone_verification_completed, setShouldShowPhoneNumberOTP, setIsForcedToExitPnv } = ui;
+    const {
+        is_phone_verification_completed,
+        setShouldShowPhoneNumberOTP,
+        setIsForcedToExitPnv,
+        should_show_phone_number_otp,
+    } = ui;
     const { trackPhoneVerificationEvents } = usePhoneVerificationAnalytics();
 
+    const getSubformName = () => {
+        if (is_at_otp_verification) {
+            return should_show_phone_number_otp ? 'verify_phone_otp_screen' : 'verify_email_screen';
```

Comment:
what if its in confirm your phone number screen?

Reply:
it will return 'verify_phone_screen' if user is not in `otp_verification` screen🙏 
![Screenshot 2024-10-04 at 3 22 33 PM](https://github.com/user-attachments/assets/d5a93231-67db-455b-bdf0-39b4676c03e9)